src/main/java/de/tum/in/www1/artemis/domain/enumeration/Rank.java

```diff
@@ -0,0 +1,8 @@
+package de.tum.in.www1.artemis.domain.enumeration;
+
```

Comment:
Maybe rename this to `AchievementRank` so that one can see directly that this belongs to achievements :)

Reply:
Good point :) done so